test/unit/vl-select.test.html

```diff
@@ -399,8 +408,48 @@
           done();
         });
       });
+
+      test('de select placeholders bevatten default waarden', (done) => {
+        const select = fixture('vl-select-auto-dress-fixture');
+        awaitUntil(() => select._dressed).then(() => {
+          assert.equal(vl.i18n.i18n['select.no_more_options'], 'Geen resterende opties gevonden');
+          assert.equal(vl.i18n.i18n['select.placeholder_value'], 'Selecteer item');
+          assert.equal(vl.i18n.i18n['select.no_results'], 'Geen resultaten gevonden');
+          assert.equal(vl.i18n.i18n['select.search_placeholder_value'], 'Zoek item');
```

Comment:
@coemans zou het niet interessanter zijn als de componenten hun messages zelf kunnen aanleveren aan de translation "service" ipv. dit allemaal in de vl-core (of waar dit zich dan ook exact bevindt) te hebben steken?

Reply:
Ja, ik begrijp ook nog niet helemaal hoe elke instantie van een select component zijn eigen message nu dan krijgt, als die vertalingen in een global object gestoken worden ...

Reply:
check heb daar gisteren ook naar zitten zoeken maar vond de link niet tussen de initialisatie van de i18n vertalingen en het uiteindelijk zetten van de vertaling, @coemans  hoe gaat dit juist in zijn werk?

Reply:
Ik vermoed op het moment dat de 'dress' wordt aangeroepen dattem die vertaling daar gaat uitlezen of zo. En omdat wij dan telkens een globale variabele aanpassen en dan direct daarna die component dressen, heeft die telkens 'de juiste'. Brak systeem, maar als @coemans daar de code van AIV wil houden ...

Reply:
Zie lib/select.js ook @stefanborghys

Reply:
De vraag is hier ondertussen al beantwoord. Inderdaad voorlopig zo dicht mogelijk bij AIV blijven, de voordelen wegen ruim op tegen de nadelen en voorlopig zijn de nadelen nog eenvoudig op te lossen door eigen code.

Los daarvan zou het interessant zijn om centraal uitbreidingen te voorzien, maar centraal zijn ze daar ook terecht voorzichtig mee om geen negatieve invloed te hebben op andere afnemers. Er lopen momenteel gesprekken om naar een centrale gezamenlijke webcomponenten codebase te groeien wat ons in de toekomst meer vrijheid zal geven om centraal grote wijzigingen door te voeren waar nodig.